.github/workflows/quality.yml

```
@@ -15,21 +15,16 @@ jobs:
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile
+        run: bun install --frozen-lockfile
 
       - name: Run lint
-        run: pnpm run lint
+        run: bun run lint
 
       # - name: Run tests
-      #   run: pnpm run test 
\ No newline at end of file
+      #   run: bun run test 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)